github/resource_github_actions_organization_secret.go

```diff
@@ -72,6 +71,7 @@ func resourceGithubActionsOrganizationSecret() *schema.Resource {
 				},
 				Set:         schema.HashInt,
 				Optional:    true,
+				ForceNew:    true,
```

Comment:
I don't think this is the right approach, both because it's [not necessary](https://docs.github.com/en/enterprise-cloud@latest/rest/actions/secrets?apiVersion=2022-11-28#set-selected-repositories-for-an-organization-secret) and because it takes the provider further away from the current idiomatic pattern for secrets.